src/routes/sign-in/sign-in-route.ts

```diff
@@ -59,3 +61,16 @@ export default {
         },
     ],
 } as RouteConfig;
+
+export const kbSSORoute = {
+    path: '/checkauth.jsp',
+    meta: {
+        excludeAuth: true,
+        isSignInPage: true,
+    },
+    props: route => ({
+        query: route.query,
+        nextPath: route.query.nextPath || '/',
+    }),
+    component: KB_SSO,
+} as RouteConfig;
```

Comment:
서비스 규모가 커지면서 라우트 path 를 기억하기 어려울 것으로 판단하여, 라우터 name들을 constant로 관리하고 있습니다.
그래서, 위의 `SIGN_IN_ROUTE` 에 작성해주신 이름을 라우트 name에 주고,
앞으로 해당 라우터로 들어와야 하는 경우에는 `SIGN_IN_ROUTE.KB._NAME `을 통해서만 들어오도록 하면 좋을 것 같습니다!
```suggestion
export const kbSSORoute = {
    path: '/checkauth.jsp',
    name: SIGN_IN_ROUTE.KB._NAME,
    meta: {
        excludeAuth: true,
        isSignInPage: true,
    },
    props: route => ({
        query: route.query,
        nextPath: route.query.nextPath || '/',
    }),
    component: KB_SSO,
} as RouteConfig;
```